sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor.cs

```diff
@@ -1296,8 +1303,11 @@ protected virtual LastEnqueuedEventProperties ReadLastEnqueuedEventProperties(st
         /// <returns>The set of identifiers for the Event Hub partitions.</returns>
         ///
         protected virtual async Task<string[]> ListPartitionIdsAsync(EventHubConnection connection,
```

Comment:
No need for `async` here now.  We can collapse down to an expression body and just return the `Task.FromResult`.